system_files/tmp/packages.json

```diff
@@ -37,34 +34,22 @@
                 "gvfs-mtp",
                 "gvfs-smb",
                 "imsettings-gsettings",
-                "initial-setup-gui",
-                "metacity",
                 "nemo-fileroller",
                 "nemo-image-converter",
                 "nemo-preview",
                 "nemo-python",
                 "nm-connection-editor",
-                "paper-icon-theme",
-                "pipewire-alsa",
+                "papirus-icon-theme",
                 "pipewire-pulseaudio",
 		"power-profiles-daemon",
-                "powerline",
                 "qgnomeplatform-qt5",
                 "redshift-gtk",
                 "sane-backends-drivers-scanners",
                 "setroubleshoot",
-                "simple-scan",
                 "slick-greeter",
                 "slick-greeter-cinnamon",
                 "system-config-printer",
-                "tmux",
-                "tmux-powerline",
-                "vim-powerline",
-                "xawtv",
-                "xdg-user-dirs-gtk",
-                "xed",
-                "xfburn",
-                "xreader"
+                "xdg-user-dirs-gtk"
             ]
```

Comment:
rationale for each change:

abrt: useful for reporting errors to fedora, not sure how useful reports from this custom image would be
eom: eog is available via flathub
initial-setup-gui: never used since this image is accessible only via a rebase
metacity: as far as I can tell, support for metacity has been dropped by mint
paper-icon-theme replaced with papirus. paper is missing many icons
pipewire-alsa: alsa is not used anymore
powerline, tmux: opinionated, can be relayered if needed
xawtv: what even is this?
xed: text editors are available via flathub
xfburn: can be relayered if needed
xreader: document reader is available via flathub